app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntryViewRenderer.kt

```diff
@@ -1,11 +1,21 @@
 package org.simple.clinic.bp.entry
 
+import org.simple.clinic.bp.entry.OpenAs.New
 import org.simple.clinic.mobius.ViewRenderer
 
 class BloodPressureEntryViewRenderer(
     val ui: BloodPressureEntryUi
 ) : ViewRenderer<BloodPressureEntryModel> {
   override fun render(model: BloodPressureEntryModel) {
-    // Do nothing!
+    renderOpenAs(model.openAs)
```

Comment:
Rename method to `setupUi`?

Reply:
I can't think of a good name, this essentially the intention of `set up the ui based on what it was opened for`.

Reply:
Will fix.